### ISSUE TITLE
kubectl-moco cannot authenticate to a GKE cluster

### ISSUE THREAD

**Issue by pbuszka** [cybozu-go/moco#320]
**Describe the bug**
When trying to access moco mysql cluster deployed ona GKE kubectl moco fails with 
```
kubectl moco -n foo mysql -it test
Error: no Auth Provider found for name "gcp"
no Auth Provider found for name "gcp"
```

**Environments**

kubectl-moco version 0.10.6

kubectl version
Client Version: version.Info{Major:"1", Minor:"20+", GitVersion:"v1.20.8-dispatcher", GitCommit:"283881f025da4f5b3cefb6cd4c35f2ee4c2a79b8", GitTreeState:"clean", BuildDate:"2021-09-14T05:12:27Z", GoVersion:"go1.15.13", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"20+", GitVersion:"v1.20.10-gke.301", GitCommit:"17ad7bd6afa01033d7bd3f02ce5de56f940a915d", GitTreeState:"clean", BuildDate:"2021-08-24T05:18:54Z", GoVersion:"go1.15.15b5", Compiler:"gc", Platform:"linux/amd64"}
- OS: Ubuntu 20.04

**To Reproduce**
1. Create a cluster on GKE (Google Kubernets Engine)
2. Install a moco example mysql cluster as per installation instruction
3. run any kubectl moco which access cluster i.e
kubectl moco -n foo mysql -it test

**Expected behavior**
Ability to use kubectl-moco with a GKE cluster

**Additional context**
The issue probably comes down to the problem mentioned in this issue 
https://github.com/kubernetes/client-go/issues/839
and proposed solution to inlude appropriate auth libs
https://github.com/kubernetes/client-go/blob/master/examples/out-of-cluster-client-configuration/main.go#L33-L40

Unfortunatly, I don't know go good enough to propose a meaningful PR

**Comment by ymmt2005** [comment]
Thank you for the report.  I'll take a look.

**Comment by zoetrope** [comment]
We need `_ "k8s.io/client-go/plugin/pkg/client/auth"` ?

**Comment by ymmt2005** [comment]
Sure. Those lines are included in the scaffold generated by kubebuilder.

```go
import (
        // Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
        // to ensure that exec-entrypoint and run can make use of them.
        _ "k8s.io/client-go/plugin/pkg/client/auth"
```

**Comment by masa213f** [comment]
@pbuszka 
Thanks for the report!
I have released the fixed version as MOCO v0.10.7. If you like, please try it.
